src/lib/services/user-service.ts

```
@@ -0,0 +1,156 @@
+            data: {
+                deletedAt: new Date(),
+                isActive: false,
+            },
+        });
+    }
+
+    /**
+     * Verify user credentials
+     */
+    static async verifyCredentials(email: string, password: string) {
+        const user = await this.getUserByEmail(email);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)